components/AuthButton.tsx

```diff
@@ -27,4 +27,4 @@ export default async function AuthButton() {
       Login
     </Link>
   );
-}
```

Comment:
Revert this. It will look like you were the last one to touch this file when it was only a white space issue.